File: proyecto-backend-nexu/src/main/java/com/nexu/Plans/dto/PlansResponseDto.java
package com.nexu.Plans.dto;

import com.nexu.Places.domain.Places;
import com.nexu.Plans.domain.VisibilityPlans;
import com.nexu.Users.domain.Users;
import lombok.Data;

import java.time.LocalDate;
import java.time.LocalDateTime;
import java.util.Set;

@Data
public class PlansResponseDto {

    private Long id;
    private String name;
    private String description;
    private LocalDate startDate;
    private LocalDate endDate;
    private LocalDateTime createdDate;
    private VisibilityPlans visibility;

    private Set<Long> usersId;
    private Set<Long> placesId;
    private Long creatorId;
}

File: proyecto-frontend-mobile-nexu3/src/interfaces/plans/PlansResponseDto.ts
export interface PlansResponseDto {
  id: number;
  name: string;
  description?: string;
  startDate?: string; // ISO string (YYYY-MM-DD)
  endDate?: string;   // ISO string (YYYY-MM-DD)
  createdDate?: string; // ISO string (YYYY-MM-DDTHH:mm:ss)
  visibility: VisibilityPlans;
  usersId: number[];
  placesId: number[];
  creatorId: number;
}

import { VisibilityPlans } from './PlansRequestDto'; 